src/sentry/static/sentry/app/views/onboarding/project/platformpicker.jsx

```diff
@@ -19,112 +26,220 @@ class PlatformPicker extends React.Component {
     showOther: PropTypes.bool,
   };
 
-  static defaultProps = {showOther: true};
+  static defaultProps = {
+    showOther: true,
+  };
 
-  constructor(...args) {
-    super(...args);
+  constructor(props, ...args) {
+    super(props, ...args);
     this.state = {
-      tab: allCategories[0].id,
-      filter: (this.props.platform || '').split('-')[0],
+      category: PLATFORM_CATEGORIES[0].id,
+      filter: props.platform?.split('-')[0],
     };
   }
 
-  logSearch = _.debounce(() => {
+  get platformList() {
+    const {category} = this.state;
+    const currentCategory = categoryList.find(({id}) => id === category);
+
+    const subsetMatch = ({id}) => id.includes(this.state.filter.toLowerCase());
+    const categoryMatch = platform =>
+      category === 'all' || currentCategory.platforms.includes(platform.id);
+
+    const filtered = flattenedPlatforms.filter(
+      this.state.filter ? subsetMatch : categoryMatch
+    );
+
+    return this.props.showOther ? filtered : filtered.filter(({id}) => id !== 'other');
+  }
+
+  logSearch = debounce(() => {
     if (this.state.filter) {
       analytics('platformpicker.search', {
         query: this.state.filter.toLowerCase(),
-        num_results: this.getPlatformList().length,
+        num_results: this.platformList.length,
       });
     }
   }, 300);
 
-  getPlatformList = () => {
-    const subsetMatch = ({id}) => id.includes(this.state.filter.toLowerCase());
-    let filtered;
-
-    if (this.state.filter) {
-      filtered = flattenedPlatforms.filter(subsetMatch);
-    } else {
-      const {tab} = this.state;
-      const currentCategory = categoryList.find(({id}) => id === tab);
-      const tabSubset = flattenedPlatforms.filter(platform => {
-        return tab === 'all' || currentCategory.platforms.includes(platform.id);
-      });
-      filtered = tabSubset.filter(subsetMatch);
-    }
+  @keydown('/')
+  focusSearch(e) {
```

Comment:
Do we need to bind `this` here?

Reply:
It doesn't seem to be required (this functionality works).